### ISSUE TITLE
[Feature Request] v-menu expose close event

### ISSUE THREAD

**Issue by miranda-elliott** [vuetifyjs/vuetify#7145]
### Problem to solve
Want to call method on v-menu close.

### Proposed solution
Expose close event for v-menu.

<h4>Example usage</h4>

[Code pen](https://codepen.io/anon/pen/WBeoeW)

Want to save multiple selected months on v-menu close when v-menu is closed by clicking outside of v-menu instead of requiring user to click OK. Tried adding `@click:month="$refs.menu.save(date)"` to v-date-picker, which makes v-menu close on month selection. Would like to add `@close="$refs.menu.save(date)"` to v-menu to accomplish this.

If there is an existing workaround, please let me know. Thank you!

**Comment by KaelWD** [comment]
`@input`

**Comment by miranda-elliott** [comment]
When I add `@input="$refs.menu.save(date)"` to v-menu the menu won't open. When I add `@input="$refs.menu.save(date)"` to v-date-picker the menu closes on month selection, as it did with `@click:month="$refs.menu.save(date)"`. How should I be using it?

**Comment by thrazu** [comment]
The return-value prop could be an option.

**Comment by zeroarst** [comment]
I started using Vuetify this week and I am trying v-menu component. I found there is a way to monitor the open/close event. just use `v-model`:
```
<v-menu offset-y v-model="menuOpened">
      ....
</v-menu>
```

I found this by looking at Vue's event's section in dev tool in chrome.
return-value is also an option.
![2020-02-28 10_48_42-](https://user-images.githubusercontent.com/1834837/75500311-4a456c00-5a18-11ea-99ce-2da6f4c4c018.png)
![2020-02-28 10_48_55-Sniip](https://user-images.githubusercontent.com/1834837/75500317-4d405c80-5a18-11ea-9494-d77e68692eab.png)
